Documentation/using-rkt-with-systemd.md

```diff
@@ -172,32 +155,32 @@ To make socket activation work, you need to add to your application manifest a [
 }
 ```
 
-Then you will need a pair of service and socket unit files. The socket unit file will have the same port you've set in the image manifest of your app and the service unit file will just run `rkt`:
+Then you will need a pair of `.service` and `.socket` unit files. The socket unit file will have the same port you've set in the image manifest of your app, and the service unit file will run `rkt`:
 
 ```
-# my-socket-activated-app.service
+# my-socket-activated-app.socket
 [Unit]
-Description=My socket-activated app
+Description=My socket-activated app's socket
 
-[Service]
-ExecStart=/usr/bin/rkt run myapp.com/my-socket-activated-app:v1.0
-KillMode=mixed
+[Socket]
+ListenStream=8080
 ```
 
 ```
-# my-socket-activated-app.socket
+# my-socket-activated-app.service
 [Unit]
-Description=My socket-activated app's socket
+Description=My socket-activated app
 
-[Socket]
-ListenStream=8080
+[Service]
+ExecStart=/usr/bin/rkt run myapp.com/my-socket-activated-app:v1.0
+KillMode=mixed
 ```
 
-Finally you start the socket unit:
+Finally, start the socket unit:
 
 ```
-# systemctl start my-socket-activated-app.socket
-# systemctl status my-socket-activated-app.socket
+$ systemctl start my-socket-activated-app.socket
```

Comment:
No, I didn't distinguish between user and root shell prompts in the style
guide. I was aiming with these changes to make the prompts consistent. I'll
come back and consider the user/root prompt thing later, docs-wide.
Acceptable?

On Tuesday, February 2, 2016, Jonathan Boulle <notifications@github.com>
wrote:

> In Documentation/using-rkt-with-systemd.md
> <https://github.com/coreos/rkt/pull/2066#discussion_r51601268>:
>
> >
> >  ```
> > -# systemctl start my-socket-activated-app.socket
> > -# systemctl status my-socket-activated-app.socket
> > +$ systemctl start my-socket-activated-app.socket
>
> hmm I see our style guide does NOT differentiate between root/nonroot
> commands....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2066/files#r51601268>.
>